app/data-services/usersService/usersService.js

```diff
@@ -1,22 +1,38 @@
 const q = require('q');
+var winston = require('winston');
 const NotFoundException = require('../../models/exceptions/NotFoundException');
 
+//logger file creation
```

Comment:
Can this rather be extracted into a logging service so that this doesn't have to be created in every file that needs logging